src/components/Playbooks/Runs/Actions/PlaybookRunsActionItem.tsx

```diff
@@ -34,10 +35,23 @@ export default function PlaybookRunsActionItem({
         </div>
         <div className={`text-xs flex flex-row gap-1 items-center`}></div>
       </div>
-      <div className="flex flex-col">
+      <div className="flex flex-row gap-2">
         <div className="text-sm  text-gray-600">
           <Age from={action.start_time} to={action.end_time} />
         </div>
+        {/* only show, if scheduled time is in the future */}
+        {action.status === "sleeping" &&
+          action.scheduled_time &&
+          dayjs().isBefore(dayjs(action.scheduled_time)) && (
+            <div className="text-sm text-gray-600">
+              Resumes in
```

Comment:
Seems to be a typo here

![image](https://github.com/flanksource/flanksource-ui/assets/13419481/f171b06d-509d-4351-bcc1-7de392b44cb6)